tcms/core/ajax.py

```diff
@@ -382,7 +384,7 @@ def update_bugs_to_caseruns(request):
                 for bug in bugs:
                     if bug.case_run_id == run.pk:
                         run.remove_bug(bug.bug_id, run.pk)
-    except Exception as error:
+    except Exception as error:  # pylint: disable=broad-except
```

Comment:
This needs to change. `remove_bug` doesn't raise and `add_bug` will raise `ValueError` from what I can see. If there are other types of exceptions we need to know what they are not blindly ignore them.